src/Command/Cache/LineItemCacheWarmerCommand.php

```diff
@@ -117,14 +117,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
             /** @var LineItem $lineItem */
             foreach ($lineItemCollection as $lineItem) {
-                $id = (int)$lineItem->getId();
+                $id = $lineItem->getId();
                 $lineItemCacheKey = $this->lineItemCacheIdGenerator->generate($id);
 
                 $this->resultCacheImplementation->delete($lineItemCacheKey);
                 $this->lineItemRepository->findOneById($id);
 
                 $this->logger->info(
-                    sprintf('Result cache for Line Item Id %d have been successfully warmed up.', $id),
+                    sprintf("Result cache for Line Item with id = '%s' have been successfully warmed up.", (string)$id),
```

Comment:
Not sure what suites best but is also possible to use $id->jsonSerialize() instead of casting.

Reply:
Agree with Luis on possible usage, but IMO - doesn't really matter